### PR TITLE
Update readme with SCA details

### PR DESCRIPTION
**Pull request by Saarett**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,11 @@
 [![slack-community](https://img.shields.io/badge/Slack-4A154B?style=plastic&logo=slack&logoColor=white)](https://slack.bridgecrew.io/)
  
 
-**Checkov** is a static code analysis tool for infrastructure-as-code.
+**Checkov** is a static code analysis tool for infrastructure-as-code and it is also a software-composition-analysis tool for open-source images and packages.
```
Comment (tsmithv11) [inline README.md:16]: Using dashes is pretty uncommon and we scan closed source images. Suggest:
`**Checkov** is a static code analysis tool for infrastructure as code (IaC) and also a software composition analysis (SCA) tool for images and open source packages.`

File: README.md [hunk 2]
```diff
@@ -13,9 +13,11 @@
...
 
 It scans cloud infrastructure provisioned using [Terraform](https://terraform.io/), [Terraform plan](docs/7.Scan%20Examples/Terraform%20Plan%20Scanning.md), [Cloudformation](docs/7.Scan%20Examples/Cloudformation.md), [AWS SAM](docs/7.Scan%20Examples/AWS%20SAM.md), [Kubernetes](docs/7.Scan%20Examples/Kubernetes.md), [Helm charts](docs/7.Scan%20Examples/Helm.md),[Kustomize](docs/7.Scan%20Examples/Kustomize.md), [Dockerfile](docs/7.Scan%20Examples/Dockerfile.md),  [Serverless](docs/7.Scan%20Examples/Serverless%20Framework.md), [Bicep](docs/7.Scan%20Examples/Bicep.md), [OpenAPI](docs/7.Scan%20Examples/OpenAPI.md) or [ARM Templates](docs/7.Scan%20Examples/Azure%20ARM%20templates.md) and detects security and compliance misconfigurations using graph-based scanning.
+
+It scans open source packages and images for [Common Vulnerabilities and Exposures](https://cve.mitre.org/)
```
Comment (tsmithv11) [inline README.md:20]: Can we point to docs instead of MITRE like the other links in this section?

**Review by tsmithv11 — COMMENTED** [review]
Good additions. Made a few suggestions.

File: docs/7.Scan Examples/Sca.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+published: true
+title: SCA scanning
+nav_order: 20
+---
+
+# SCA scanning
+Checkov is an SCA (Software Composition Analysis) tool. This means it scans package files closed source images for Common Vulnerabilities and Exposures (CVEs).
```
Comment (taysmith-test) [inline docs/7.Scan Examples/Sca.md:9]: Recommend: `Checkov is an SCA (Software Composition Analysis) tool. This means it scans package files and container images for Common Vulnerabilities and Exposures (CVEs).`

File: docs/7.Scan Examples/Sca.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+You can [Find here](https://docs.bridgecrew.io/docs/open-source-vulnerability-scanning) the full list of the supported package manager types.
```
Comment (taysmith-test) [inline docs/7.Scan Examples/Sca.md:11]: `You can find the full list of the supported package manager types [here]((https://docs.bridgecrew.io/docs/open-source-vulnerability-scanning).`

**Review by taysmith-test — CHANGES_REQUESTED** [review]
Looks good! Added two suggestions

**Pull request merged**